PopPUNK/visualise.py

```diff
@@ -84,12 +84,19 @@ def get_options():
                         'minimum spanning tree',
                         default=None,
                         type = str)
+    iGroup.add_argument('--recalculate-distances',
+                        help='Recalculate pairwise distances rather than read them from a file',
+                        default=False,
+                        action = 'store_true')
     iGroup.add_argument('--network-file',
                         help='Specify a file to use for any graph visualisations',
                         type = str)
     iGroup.add_argument('--display-cluster',
                         help='Column of clustering CSV to use for plotting',
                         default=None)
+    iGroup.add_argument('--use-partial-query-graph',
+                        help='File listing sequences in partial query graph after assignment',
+                        default=None)
```

Comment:
Similar note to arg above

Reply:
I think this is helpfully consistent with the `--save-partial-query-graph` of `assign.py`.